bindings_generator/src/api.rs

```diff
@@ -3,12 +3,14 @@ use serde_json;
 
 pub struct Api {
     pub classes: Vec<GodotClass>,
+    pub api_underscore: Vec<String>,
```

Comment:
This could use a `std::collections::HashSet<String>` instead since order and random access is not required.

Reply:
Yeah, I was just throwing something in here and hadn't used the Set form of HashSet (always using HashMap) so unfamiliar.